article/Name-YEAR.md

```diff
@@ -151,36 +151,40 @@ analysis of the results.
 Figure 3d. However, when using the same scale for the two graphs, we can see
 that the k-values in the case of high clumping show much less variation than
 when the lower clumping is used.
 
 The simulations performed to study the relationship between the population
 dynamics of the hosts and the generalist parasitoids are also very similar to
-the ones done by Hassell. This show a density dependant relationship between the
+the ones done by Hassell (@fig:figure4). This shows a density dependant relationship between the
 two populations, where the natural enemy regulates the host population until
 they both reach a stable equilibrium. This is also the conclusion when we
-observe the relation between the two populations in Figures 3a and 3c.
+observe the relation between the two populations in @fig:figure3 (a) and (c).
 
 ### Stochastic model
 
+When the models include stochasticity, the resulting population dynamics between hosts and parasites are much more different than when the models are deterministic. Overall, the results obtained in the replications are pretty similar than the ones from Hassell, except for @fig:figure6 (d), where the correlation between the mortality and the host density is not nearly as strong as in the original paper. Whether we look at the model for the specialist parasitoids or the one for the generalist parasitoids, both show irregular oscillations in the host-parasitoid population dynamics (@fig:figure5 and @fig:figure6, (a), (b) and (c)). The addition of stochastic parameters prevents the stabilization of host populations and makes it more difficult to identify parasitoids as a density-dependent control factor. The relationship between the k-values and the host density are similar in Hassell's publication and in ours (except in @fig:figure6 (d)). The simulations done in this paper tend to have a determination coefficient (R^2) a little higher than the one found by Hassell, but they are generally really close.
+
+Because the inclusion of stochastic parameters in the population dynamics causes variability in the outputs, the results from two successive simulations can be very different. In order to account for this variability and to show how it can affect the population dynamics of the hosts and parasites, @fig:figure7 and @fig:figure8 show the extent of the variation of the correlation coefficient ($r$) obtained in 5000 different simulations. A dotted line was added to represent the value of the correlation coefficient ($r$) that came out of the deterministic models. The values of r vary greatly for every stochastic parameter, but the one thing that stays constant is that in every case, the mean value of $r$ is lower in the stochastic models than in the deterministic models. This is in agreement with Hassell's results, and shows that stochasticity makes it harder to see the density dependence effect of the parasites, whether they are generalist or specialist.
```

Comment:
p-e simplifier la phrase "In order to account for this variability and to show how it can affect the population dynamics of the hosts and parasites, @fig:figure7 and @fig:figure8 show the extent of the variation of the correlation coefficient ($r$) obtained in 5000 different simulations."

Genre, "we added figure 7-8 to show the extent of the variation in the correlation coefficient obtained in 5000 different simulations"